core/src/autogluon/core/metrics/__init__.py

```diff
@@ -95,36 +125,46 @@ def add_alias(self, alias):
 
     @property
     def greater_is_better(self) -> bool:
-        """Return whether the score is greater the better.
-
-        We use the stored `sign` object to decide the property.
+        """
+        Return whether the score returned by scorer(...) is in greater_is_better format.
+        By default, all Scorers are in greater_is_better format.
 
         Returns
         -------
-        flag
+        flag :
             The "greater_is_better" flag.
         """
+        return True
+
+    @property
+    def greater_is_better_internal(self) -> bool:
+        """
+        Return whether the inner self.score_func returns metric values in greater_is_better format.
+
+        We use the stored `sign` variable to decide the property.
+        Users should not need to access this property during normal usage.
```

Comment:
Since it is not supposed to be used by the users, should we make it private (starts with underscore)?

Reply:
It isn't quite not supposed to be used by users as it is that it would be a very odd situation where they would use this. The only time I use it is when logging metric values, I'll add a `-` in front of the metric name if `greater_is_better != greater_is_better_internal`. I think it is ok to stay as is.

Reply:
Other reasons to use it is when interacting with non-AG implementations of the metric such as models that don't support Scorer, so that you can convert the values to a consistent representation between the metrics.